src/language-service/src/schemas/integrations/triggers.ts

```diff
@@ -552,6 +553,20 @@ interface WebhookTrigger {
   variables?: Data;
 }
 
+interface WebOSTvTrigger {
```

Comment:
This trigger belong to the webos integration and is not part of the core trigger. Please split it out of this file.

Reply:
Hm, isn't this similar to the MQTTTrigger which is also in this file?

Reply:
I've attempted to move it, please check. Not sure what should be the proper file structure.